app/exporters/patients_with_history_exporter.rb

```diff
@@ -33,6 +33,27 @@ def self.csv(*args)
     new.csv(*args)
   end
 
+  def self.csv_enumerator(*args)
+    new.csv_enumerator(*args)
+  end
+
+  def csv_enumerator(patients, display_blood_sugars: true)
+    @display_blood_sugars = display_blood_sugars
+    summary = MaterializedPatientSummary.where(patient: patients)
+
+    Enumerator.new do |yeilder|
+      yeilder << timestamp
+      yeilder << measurement_headers
+      yeilder << csv_headers
+
+      summary.in_batches(of: BATCH_SIZE).each do |batch|
+        batch.each do |patient_summary|
+          yeilder << csv_fields(patient_summary)
+        end
+      end
+    end
+  end
```

Comment:
[nonblocking]
...continuing from the previous comment, the change needed to make this yield CSV strings rather than arrays:
```suggestion
  def csv_enumerator(patients, display_blood_sugars: true)
    @display_blood_sugars = display_blood_sugars
    summary = MaterializedPatientSummary.where(patient: patients)

    Enumerator.new do |yeilder|
      yeilder << timestamp.to_csv
      yeilder << measurement_headers.to_csv
      yeilder << csv_headers.to_csv

      summary.in_batches(of: BATCH_SIZE).each do |batch|
        batch.each do |patient_summary|
          yeilder << csv_fields(patient_summary).to_csv
        end
      end
    end
  end
```

There's probably a better way, but can't think of one right now.

Reply:
Also typo: `yielder` not `yeilder`